clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java

```diff
@@ -102,6 +103,18 @@
   @ClassRule
   public static final TestCounter TEST_COUNTER = new TestCounter();
 
+  /**
+   * The tests in this class are failing on Windows.  There is a windows bug in the Galvan code.
+   * This is a temporary fix until the Galvan bug is resolved.  When the code below is run
+   * on a Windows OS all the tests will be skipped.
+   * The Galvan issue is Terracotta-OSS/galvan#61
+  **/
+  @BeforeClass
+  public static void checkOS()
+  {
```

Comment:
shoot!  sorry about the formatting.  I am removing this method, as I fixed the bug in Galvan.